cryptography/src/bls12381/primitives/group.rs

```diff
@@ -189,14 +189,18 @@ impl Scalar {
         Self(ret)
     }
 
-    /// Sets the scalar to be the provided integer.
-    pub fn set_int(&mut self, i: u32) {
+    /// Creates a new scalar from the provided integer.
+    fn from(i: u64) -> Self {
```

Comment:
Changed this to return `Self` rather than operating over `&mut self` (to make it clearer that whatever was there is clobbered)